src/main/java/net/openhft/chronicle/core/onoes/ExceptionKey.java

```diff
@@ -82,7 +89,8 @@ public String toString() {
         if (throwable != null)
             throwable.printStackTrace(new PrintWriter(sw));
         return "ExceptionKey{" +
-                "level=" + level +
+                "nanoTimestamp=" + nanoTimestamp +
```

Comment:
of two minds whether to include this in the toString, and if we do, perhaps we format it?